### PR TITLE
Add use cases to DG and JM's information to AboutUs

### PR DESCRIPTION
**Pull request by jmleong666**
_Linked issues_: #5, #6. Add use cases for purge and clear commands, as well as a few NFRs to DG. Add JM's information to AboutUs and JM's portfolio page.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -390,13 +390,36 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     
         Use case resumes at step 2. 
 
+**Use case: Clear past interactions**
+
+**MSS**
+1.  User enters the clear command.
+2.  StonksBook clears the GUI.
```
Comment (AaronnSeah) [inline docs/DeveloperGuide.md]: I think that clears the GUI might not be clear enough as the the contact list is also a GUI. You could consider chatbot GUI or something that it is clearer?

Comment (AaronnSeah) [reply docs/DeveloperGuide.md]: or clear the chat messages in the chatbot GUI

Comment (jmleong666) [reply docs/DeveloperGuide.md]: Yeah I'll do a quick fix.

**Review by AaronnSeah — APPROVED** [review]
LGTM

**Pull request merged**